app/javascript/utils/Patient.js

```diff
@@ -61,4 +67,36 @@ function formatRace(patient) {
   return raceArray.length === 0 ? '--' : raceArray.join(', ');
 }
 
-export { formatName, formatNameAlt, formatPhoneNumber, formatRace };
+function isMinor(date) {
+  return moment(date, 'YYYY-MM-DD').isAfter(moment().subtract(18, 'years')); 
+}
```

Comment:
Please add function comment, include that the param must be of form `'YYYY-MM-DD'`